src/components/sbb-divider/sbb-divider.tsx

```diff
@@ -0,0 +1,35 @@
+import { Component, h, Prop } from '@stencil/core';
+import { InterfaceSbbDividerAttributes } from './sbb-divider.custom.d';
+
+/**
+ * @slot unnamed - Use this to document a slot.
+ */
+
+@Component({
+  shadow: true,
+  styleUrl: 'sbb-divider.scss',
+  tag: 'sbb-divider',
+})
+export class SbbDivider {
+  /** Appearance property for displaying the component in dark mode */
+  @Prop() public appearance?: InterfaceSbbDividerAttributes['appearance'] = 'primary';
```

Comment:
This should be negative property: https://github.com/lyne-design-system/lyne-components/blob/master/CODING_STANDARDS.md#use-negative-property-if-a-component-has-a-color-negative-specification

Reply:
Refactored.

Reply:
Fixed 👍